packages/base-driver/test/basedriver/capabilities-specs.js

```diff
@@ -434,4 +435,43 @@ describe('caps', function () {
       findNonPrefixedCaps({alwaysMatch, firstMatch}).should.eql(['nonStandardOne', 'nonStandardTwo', 'nonStandardThree', 'nonStandardFour', 'nonStandardFive']);
     });
   });
+
+  describe('#promoteAppiumOptions', function () {
+    const appiumCaps = {
+      'appium:platformVersion': '14.4',
```

Comment:
I would also add one option with a different vendor prefix to make sure it's preserved